AppKit/CPTrackingArea.j

```diff
@@ -0,0 +1,143 @@
+/*
+ * CPTrackingArea.j
+ * AppKit
+ *
+ * Created by Didier Korthoudt.
+ * Copyright 2015, Cappuccino Project.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+@import <Foundation/Foundation.j>
+@import "CPView.j"
+
+/* @group CPTrackingAreaOptions */
+@typedef CPTrackingAreaOptions
+CPTrackingMouseEnteredAndExited     = 1 << 1;
+CPTrackingMouseMoved                = 1 << 2;
+CPTrackingCursorUpdate              = 1 << 3;
+CPTrackingActiveWhenFirstResponder  = 1 << 4;
+CPTrackingActiveInKeyWindow         = 1 << 5;
+CPTrackingActiveInActiveApp         = 1 << 6;
+CPTrackingActiveAlways              = 1 << 7;
+CPTrackingAssumeInside              = 1 << 8;
+CPTrackingInVisibleRect             = 1 << 9;
+CPTrackingEnabledDuringMouseDrag    = 1 << 10;
+
+var CPTrackingAreaRectKey            = @"CPTrackinkAreaRectKey",
+    CPTrackingAreaOptionsKey         = @"CPTrackingAreaOptionsKey",
+    CPTrackingAreaOwnerKey           = @"CPTrackingAreaOwnerKey",
+    CPTrackingAreaUserInfoKey        = @"CPTrackingAreaUserInfoKey",
+    CPTrackingAreaReferencingViewKey = @"CPTrackingAreaReferencingViewKey",
+    CPTrackingAreaActualRect         = @"CPTrackingAreaActualRect";
+
+
+/*!
+ @ingroup appkit
+ 
+ A CPTrackingArea defines a region of view that generates mouse-tracking and
+ cursor-update events when the mouse is over that region.
+ */
+@implementation CPTrackingArea : CPObject
+{
+    CGRect                  _rect               @accessors(getter=rect);
```

Comment:
Let's be clearer about the coordinate system of the different rects. Please rename this to `_viewRect` and rename `_actualRect` to `_windowRect`.